
From: Paolo 'Blaisorblade' Giarrusso <blaisorblade_spam@yahoo.it>

Drop that header inclusion - I discovered this header was unused while
checking whether I can use the __set_page_dirty_nobuffers speedup suggested by
Andrew Morton.

Signed-off-by: Paolo 'Blaisorblade' Giarrusso <blaisorblade_spam@yahoo.it>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/fs/hostfs/hostfs_kern.c |    1 -
 1 files changed, 1 deletion(-)

diff -puN fs/hostfs/hostfs_kern.c~uml-drop-unused-buffer_headh-header-from-hostfs fs/hostfs/hostfs_kern.c
--- 25/fs/hostfs/hostfs_kern.c~uml-drop-unused-buffer_headh-header-from-hostfs	Thu Jan 13 15:27:07 2005
+++ 25-akpm/fs/hostfs/hostfs_kern.c	Thu Jan 13 15:27:07 2005
@@ -15,7 +15,6 @@
 #include <linux/pagemap.h>
 #include <linux/blkdev.h>
 #include <linux/list.h>
-#include <linux/buffer_head.h>
 #include <linux/root_dev.h>
 #include <linux/statfs.h>
 #include <linux/kdev_t.h>
_
